application-blog-ui/src/main/resources/Blog/RssCode.xml

```diff
@@ -163,7 +163,7 @@
       ## This is just a list of blog entries, which are detailed below.
       #foreach ($entryDoc in $entries)
         #if($xwiki.hasAccessLevel('view', ${entryDoc.fullName}))
-        &lt;rdf:li rdf:resource="$entryDoc.getExternalURL('view', "language=${entryDoc.realLanguage}")" /&gt;
+        &lt;rdf:li rdf:resource="$entryDoc.getExternalURL('view', "language=${entryDoc.realLocale.language}")" /&gt;
```

Comment:
```suggestion
        &lt;rdf:li rdf:resource="$entryDoc.getExternalURL('view', "language=${entryDoc.realLocale}")" /&gt;
```

I know the query string parameter is ``language`` but in practice it is used as a locale (you can pass for instance ``?language=fr_CA``).